tcms/issuetracker/bitbucket.py

```diff
@@ -94,7 +94,7 @@ class BitBucket(IssueTrackerType):
 
             ``api_username`` is your BitBucket username, which you use to log in.
 
-        .. info::
+        .. note::
```

Comment:
FTR there is the `.. important::` directive. I will merge as-is, if you decide it needs changing you can send another PR.